mlir/include/mlir/Dialect/Rock/IR/RockGemmWrapperInterface.td

```diff
@@ -129,9 +129,32 @@ def RockGemmWrapperInterface : OpInterface<"RockGemmWrapperInterface"> {
         /*defaultImplementation=*/[{
           $_op->setAttr($_op.getParamsAttrName(), params);
         }]
+      >,
+    InterfaceMethod<
+        /*desc=*/[{
+          Get Optional blocksize
+        }],
+        /*retType=*/"Optional<uint32_t>",
+        /*methodName=*/"getDerivedBlockSize",
+        /*args=*/(ins),
+        /*methodBody=*/"",
+        /*defaultImplementation=*/""
```

Comment:
`$_op.getDerivedBlockSize()` or something to do with `getDerivedBlockSizeAttr()` and `getZExtValue()` if you can't do that.